cli/src/cli.c

```diff
@@ -224,6 +216,32 @@ logging_init(glusterfs_ctx_t *ctx, struct cli_state *state)
     return 0;
 }
 
+static struct iobuf *
```

Comment:
I think instead of writing a new function we can directly call iobuf_get_from_small and make a function non-static.

Reply:
Will do!